### PR TITLE
feat: change PIN method

### PR DESCRIPTION
**Pull request by pedroferreira1**

### CODE REVIEW

**Comment by pedroferreira1** [comment]
That's true! It was done in the last commit. Can you check again, please? @obiyankenobi

File: src/wallet.js [hunk 1]
```diff
@@ -394,6 +394,39 @@ const wallet = {
...
+    }
+
+    const accessData = storage.getItem('wallet:accessData');
+
+    // Get new PIN hash
+    const newHash = this.hashPassword(newPin);
+    // Update new PIN data in storage
+    accessData['hash'] = newHash.key.toString();
+    accessData['salt'] = newHash.salt;
+    storage.setItem('wallet:accessData', accessData);
```
Comment (obiyankenobi) [inline src/wallet.js]: You should move this `setItem` to the end of the method, after decrypting/encrypting the `mainKey`.

File: src/wallet.js [hunk 2]
```diff
@@ -394,6 +394,39 @@ const wallet = {
     }
   },
 
+  /**
+   * Validates old PIN and change it for the new one
```
Comment (msbrogli) [inline src/wallet.js]: Typo: Validate old PIN...

File: src/wallet.js [hunk 3]
```diff
@@ -394,6 +394,39 @@ const wallet = {
...
+   *
+   * @param {string} oldPin
+   * @param {string} newPin
+   *
+   * @return {boolean} if PIN was changed
```
Comment (msbrogli) [inline src/wallet.js]: Suggestion: True if the PIN was successfully changed.

**Pull request merged**